### PR TITLE
[wolfmqtt] update to 1.19.0

### PR DESCRIPTION
**Pull request by lealem47**

### CODE REVIEW

**Comment by lealem47** [comment]
@microsoft-github-policy-service agree company="wolfSSL"

File: ports/wolfmqtt/portfile.cmake [hunk 1]
```diff
@@ -1,11 +1,9 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO wolfssl/wolfmqtt
-    REF v1.16.0
-    SHA512 30d28a37b26adacae589b9cfee9a2cd8697fa1e84ac867d08123dab5434bb19152adb0d94224656d2c1f01833d76bcd7ff63b63da4dbce0873b352e93b6e6b12
+    REF v1.19.0
```
Comment (WangWeiLin-MV) [inline ports/wolfmqtt/portfile.cmake:4]:
```suggestion
    REF "v${VERSION}"
```

**Review by WangWeiLin-MV — CHANGES_REQUESTED** [review]

**Review by WangWeiLin-MV — APPROVED** [review]
The port usage tests pass with the following triplets:
* x64-windows

**Comment by BillyONeal** [comment]
Can you explain why the CMake patch is no longer necessary (in order to force REQUIRED)? Thanks!

**Comment by lealem47** [comment]
> Can you explain why the CMake patch is no longer necessary (in order to force REQUIRED)? Thanks!
> 
> (I'm not saying that there's anything wrong with the change, I just want to make sure I understand why there is no problem should there be none :) )

Hi @BillyONeal, The patch is no longer necessary because it is already included in latest versions of wolfMQTT

File: ports/wolfmqtt/cmake-patch.diff
Comment (WangWeiLin-MV) [inline ports/wolfmqtt/cmake-patch.diff:1]: Merged in https://github.com/wolfSSL/wolfMQTT/blob/v1.19.0/CMakeLists.txt#L66-L69

**Review by WangWeiLin-MV — APPROVED** [review]

**Pull request merged**